packages/expo-updates/README.md

```diff
@@ -7,7 +7,7 @@
   </a>
 </p>
 
-The `expo-updates` module allows your app to download and manage remote updates to your application code.
+The `expo-updates` library enables your app to manage remote updates to your application code.
```

Comment:
Prefer module since we talk about Expo Modules and native modules:

```suggestion
The `expo-updates` module enables your app to manage remote updates to your application code.
```